examples/lock-app/silabs/src/LockManager.cpp

```diff
@@ -297,6 +297,10 @@ bool LockManager::Lock(chip::EndpointId endpointId, const Nullable<chip::FabricI
 bool LockManager::Unlock(chip::EndpointId endpointId, const Nullable<chip::FabricIndex> & fabricIdx,
                          const Nullable<chip::NodeId> & nodeId, const Optional<chip::ByteSpan> & pin, OperationErrorEnum & err)
 {
+    if (DoorLockServer::Instance().SupportsUnbolt(endpointId))
+    {
+        return setLockState(endpointId, fabricIdx, nodeId, DlLockState::kUnlatched, pin, err);
```

Comment:
When is the transition to the Unlock state that the spec calls for after this point?

Reply:
My understanding was that the transition required user action to complete, so this would come from a button press, so inside `AppTask.cpp`, we would have a call of `ActionCompleted`, which Schedules `UpdateClusterState` , which calls `DoorLockServer::Instance().SetLockState(1, newState) `.

Again, from my understanding, from the software side, this part of the code doesn't change.

Reply:
No, the transition is automatic.

Reply:
Added to kick off a transition to Unlock after Unlatch in the app and a cluster update in the LockManager